packages/common-types/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "@counterfactual/common-types",
```

Comment:
Why not just `/types`?

Reply:
common-types is  more descriptive. "The types that everyone has in common"

Reply:
Yeah I get that, it's that generally the commonality is implied given the namespace in which a generic package such as `types` would be placed in.

So adding `common` seems slightly redundant, but I digress.